compose.yml

```
@@ -1,14 +1,13 @@
   db:
-    image: postgres:17
+    image: mysql:8.0
     platform: linux/amd64
-    hostname: postgres
     volumes:
-      - db-data:/var/lib/postgresql/data
+      - db-data:/var/lib/mysql
     environment:
-      POSTGRES_HOST_AUTH_METHOD: trust
+      MYSQL_ALLOW_EMPTY_PASSWORD: "yes"
     ports: 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)